pkgs/development/python-modules/seabreeze/default.nix

```
@@ -0,0 +1,40 @@
+{ stdenv
+, fetchFromGitHub
+, python3Packages
+}:
+
+## Usage
+# In NixOS, simply add the `udev` multiple output to services.udev.packages:
+#   services.udev.packages = [ pkgs.python3Packages.seabreeze.udev ];
+
+python3Packages.buildPythonPackage rec {
```

This is being called from `python-packages.nix` so all the `python3Packages` are in scope so you can reference them properly.

```suggestion
buildPythonPackage rec {
```

And `buildPythonPackage` and the like (`pyusb`) can be a proper argument above.